ui/app/pages/swap/swap.utils.js

```diff
@@ -1,20 +1,21 @@
 import {
```

Comment:
Do you think it might be better to have one `utlis` file within swaps  ?

Reply:
Also answering your comment regarding duplicated code in swapsController.

I think it's a good idea, but note that I'm yet to do a cleanup there. Will consolidate all the utils + remove duplicated stuff in the next PR (already WIP in my local branch).